docs/lang/static.md

```diff
@@ -1,16 +1,84 @@
 # Static Timing
 
-By default, Calyx programs use a *latency-insensitive* model of computation.
-This means that the compiler does not track the number of cycles it takes to perform a computation or run a control operator
-In general, latency-insensitivity makes it easier to compose programs together and gives the compiler freedom to schedule operators however it wants.
-However, the generated hardware to schedule the execution may not be efficient–especially if the program can take advantage of the *latency* information.
```

Comment:
Great that we're deleting these weasel words about "may not be efficient" up front. It indeed deserves to come later, as it does now.